src/ui/FieldTable/Badge.ts

```diff
@@ -12,6 +12,9 @@ export interface IBadgeOptions extends IFieldValueOptions {
   colors: IBadgeColors;
```

Comment:
Since the `IBadgeColors` / `IBadgeColor` is used to specify the `colors` options as well as in the `parseColors`, `getColor()`, and `renderOneValue()` public methods, do you think it would be a good idea to expose these interfaces in the public doc?